Cargo.toml

```diff
@@ -12,7 +12,13 @@ repository = "https://github.com/drahnr/expander.git"
 fs-err = "2"
 proc-macro2 = "1"
 quote = "1"
-blake3 = "1.3"
+blake2 = "0.10"
+syn = { version = "1", optional = true, default-features = false }
 
 [dev-dependencies]
 baz = { path = "./tests/baz" }
+syn = { version = "1", features = ["extra-traits", "full"] }
+
+[features]
+default = ["syndicate"]
```

Comment:
This demands some README changes, right ?

Reply:
Good point!